benchmarks/workloads/absl/Dockerfile

```diff
@@ -16,7 +16,7 @@ RUN wget https://github.com/bazelbuild/bazel/releases/download/0.27.0/bazel-0.27
 RUN chmod +x bazel-0.27.0-installer-linux-x86_64.sh
 RUN ./bazel-0.27.0-installer-linux-x86_64.sh
 
-RUN git clone https://github.com/abseil/abseil-cpp.git
+RUN git clone --depth 1 https://github.com/abseil/abseil-cpp.git
 WORKDIR abseil-cpp
 RUN git checkout 43ef2148c0936ebf7cb4be6b19927a9d9d145b8f
```

Comment:
Here we fetch the specified revision, so git clone --depth 1 will not work. If you want to do this properly, you need to do something like this:

```
mkdir abseil-cpp && cd abseil-cpp && git init && git remote add origin https://github.com/abseil/abseil-cpp.git && git fetch --depth 1 origin 43ef2148c0936ebf7cb4be6b19927a9d9d145b8f && git checkout FETCH_HEAD
```

Reply:
got it . fixed and changed according to recommendation